backend/core/src/entity/User.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  CreateDateColumn,
+  UpdateDateColumn,
+  Index,
+} from "typeorm"
+
+@Entity()
+@Index("user_email_lower", { synchronize: false })
+class User {
```

Comment:
I'm ashamed to admit I just spend time chasing this, but `user` is essentially a reserved word in postgres because it's a built in function. Can we call the table something else without making life too complicated? I'd be fine with pluralizing everything for the DB rails-style, or just picking something else in this case -- whatever the consensus practice we prefer is.